tests/patch.ipynb

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
<p>I'd rather avoid having a whole new nb just to test one feature if possible</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/928/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>